### PR TITLE
Test over multiple generations

### PR DESCRIPTION
**Pull request by ghost**
With TODO for plotting the graph

### CODE REVIEW

File: tests/population_tests.c [hunk 1]
```diff
@@ -281,44 +284,75 @@ void test_population_next_generation()
...
+        /* log to csv file */
+        sprintf(str, "%f,", population_average_score(&after));
+        write(fd, str, strlen(str));
     }
+    close(fd);
+
+    /* TODO plot the csv file */
+
+
+    sprintf(commandstr,"cp %s /home/bashrc/develop/scalam/test.txt", log_file);
```
Comment (leeming) [inline tests/population_tests.c]: absolute filepath

Comment (ghost) [inline tests/population_tests.c]: ah, that was just there as a test, because the log file gets removed after use.

File: tests/population_tests.c [hunk 2]
```diff
@@ -347,8 +347,6 @@ void test_population_next_generation()
     /* TODO plot the csv file */
 
 
-    sprintf(commandstr,"cp %s /home/bashrc/develop/scalam/test.txt", log_file);
-    run_shell_command(commandstr);
 
     /* remove the log file */
     sprintf(commandstr,"rm -f %s", log_file);
```
Comment (leeming) [inline tests/population_tests.c:352]: Although I pointed out the absolute path, surely we need to make sure we can actually access the file that is produced? The next few lines remove this file

Comment (ghost) [inline tests/population_tests.c:352]: In this case I expect that the plotter will run under the TODO line and that the csv file won't be needed after that.

**Pull request merged**